pkg/serverless/appsec/httpsec/proxy.go

```diff
@@ -70,6 +70,7 @@ func (lp *ProxyLifecycleProcessor) OnInvokeStart(startDetails *invocationlifecyc
 		log.Debugf("appsec: proxy-lifecycle: Extracted event type: %v", eventType)
 	}
 
+	lp.invocationEvent = nil
```

Comment:
Nit: moving what's on line 100  to line 94 would achieve the same thing.

Reply:
Tried but still too conditional. I'll keep it this way for now.